### PR TITLE
Adds Beetle Cygne (WonderSwan)

### PR DESCRIPTION
**Pull request by televandalist**
Adds the Beetle Cygne core (WonderSwan) to bin/Cores/cores.json

### CODE REVIEW

File: bin/Cores/cores.json [hunk 1]
```diff
@@ -117,6 +117,11 @@
   "prosystem_libretro":{
     "name": "ProSystem",
     "systems": [51]
+  },
+    "mednafen_wswan_libretro":{
```
Comment (rzumer) [inline bin/Cores/cores.json]:
```suggestion
  "mednafen_wswan_libretro":{
```

**Comment by rzumer** [comment]
Not part of the review, but audio sync is awful with this core in RALR. It seems to run at full speed, but maybe it's not enough for the audio playback. This is on Ryzen 2700X and RALR compiled in Release mode. In RA it works fine.

**Comment by televandalist** [comment]
> Not part of the review, but performance is awful with this core in RALR, and I am on Ryzen 2700X with RALR compiled in Release mode. In RA it works fine.

I don't think it's just this core. The audio in a lot of cores took a dive and there's been some framerate issues here and there since a recent update. The only problem I've experienced with running Cygne is the occasional game that needs screen rotation.

**Comment by rzumer** [comment]
Apparently the system renders at 75Hz, so maybe the performance issue has something to do with my display only supporting 60Hz.

**Comment by Jamiras** [comment]
The "stuttering" appears to be caused by forcing the screen updates to occur during vsync.
https://github.com/RetroAchievements/RALibretro/blob/04955c04130a6c418c21e9585809bc0ea14def4a/src/Application.cpp#L172

https://wiki.libsdl.org/SDL_GL_SetSwapInterval
> 1 for updates synchronized with the vertical retrace

As such, the maximum render rate is the monitor's refresh rate, so the system caps out at 60 fps. While the framerate is hard to notice, the audio is very choppy. This is because the core thinks it's running at 75fps, and therefore isn't providing the audio data quick enough.

If I change the code to use "0" (which appears to be the default), the stuttering goes away. This seems to slightly improve framerates on the PSX core as well. I have yet to see any "tearing" or other issues typically associated with not using vsync.

File: bin/Cores/cores.json [hunk 2]
```diff
@@ -118,6 +118,11 @@
     "name": "ProSystem",
     "systems": [51]
   },
+  "mednafen_wswan_libretro":{
+    "name": "Beetle Cygne",
```
Comment (Jamiras) [inline bin/Cores/cores.json]: This should be "Mednafen WonderSwan" to match the string displayed in the title bar.

**Comment by leiradel** [comment]
RetroArch does something different I think. I believe it latches to vsync if ithe difference to the emulator FPS is under 10%, and adjust the audio resampler to compensate. I think it also does some more complicated stuff, checking if the emulator's FPS is a divisor or multiple of the monitor's refresh rate. All this is to provide a smooth emulation.

Not latching to vsync is fine I think, but now there's nothing to cap the speed so won't some emulators become too fast? Maybe RALibretro should use a high resolution timer to make sure the emulator runs in the correct speed.

I think however that this change will cause some video stutter.

**Pull request merged**